Automattic/pocket-casts-android
# Issues with the 'Extra Dark' theme

## Description
Fixed the Extra Dark theme issue.

Issue # 929 (https://github.com/Automattic/pocket-casts-android/issues/929)

## Testing Instructions
1. Open the Pocket Casts App
2. Go to your Profile
3. Go to Settings
4. Go to Appearance
5. Select the “Extra Dark” theme
6. Go back to Settings
7. Go to General: its theme should be Extra Dark.
8. Repeat the above for all sections of the Settings. They should be in Extra Dark. General, Storage & data use, Auto add to Up Next, Advanced, and About.

## Screenshot
![Screenshot_20230521_171006](https://github.com/Automattic/pocket-casts-android/assets/11698239/4d9171ed-d52a-4383-91dd-8e048f666369)
